fiftyone/utils/cvat.py

```diff
@@ -2523,6 +2524,11 @@ class CVATBackendConfig(foua.AnnotationBackendConfig):
         occluded_attr (None): an optional attribute name containing existing
             occluded values and/or in which to store downloaded occluded values
             for all objects in the annotation run
+        git_repo (None): the url of the git repository to link with the
+            created tasks and to which to upload annotations
+        push_to_git (True): whether to automatically push annotations to git
```

Comment:
Is there a purpose for `push_to_git`? Isn't it safe to assume that if a `git_repo` is provided then they always want their labels posted?

Reply:
It's not necessarily a given. When adding a `git_repo`, it links it to the created tasks. In the CVAT UI, this repo can be manually synced which users may want to do themselves rather than during upload/download time.

![image](https://user-images.githubusercontent.com/21222883/142894820-8726e763-a684-4160-867b-0ad8f528d7aa.png)